site/content/how-to/monitoring/dashboard.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "NGINX Plus Dashboard"
+description: "Learn how to view the NGINX Plus dashboard to see real-time metrics."
+weight: 200
+toc: true
+docs: "DOCS-000"
+---
+
+{{<custom-styles>}}
+
+## Overview
+
+The NGINX Plus dashboard offers a real-time live activity monitoring interface that shows key load and performance metrics of your server infrastructure. The dashboard is enabled by default for NGINX Gateway Fabric deployments that use NGINX Plus as the data plane. The dashboard is available on port 8765.
```

Comment:
would it be helpful to add a link to https://docs.nginx.com/nginx/admin-guide/monitoring/live-activity-monitoring/ ?

Reply:
I think that page is too NGINX Plus specific, for a user that actually has full control over configuration.